### DISCUSSION TITLE
Angular CLI: 15.1.2 - Erro na função subscribe()

### DISCUSSION THREAD

**Discussion by MayconRDO** [balta-io/forum#239] [Tira Dúvidas] [answered]
Olá Balta,

Estou fazendo o curso de Angular 8 com o Ionic 4, entendo que não estou usando a versão recomendada do curso, mas eu ficaria muito grato se você me respondesse como posso fazer para resolver um erro na função subscribe no login.page.ts

![image](https://user-images.githubusercontent.com/41632489/214397642-3c756bc5-ad23-4d2c-8573-1d7c944a8062.png)

O objeto
![image](https://user-images.githubusercontent.com/41632489/214397801-4cffc26f-9244-4fdf-a9f2-e85e360ab081.png)

**Comment by MayconRDO** [comment, marked as answer]
Favor desconsiderar o return true na linha 55.

**Reply by andrebaltieri** [reply]
Boa tarde, @MayconRDO, como vai?
No método onde você passa o `res:` o que ele espera como parâmetro?

**Reply by MayconRDO** [reply]
Olá @andrebaltieri tudo bem, 
Estou acompanhando essa aula, estou passando o objeto UserModel, mas como estou com uma versão diferente de node e angular estou com o seguindo erro:

Argument of type '(res: UserModel) => void' is not assignable to parameter of type '(value: Object) => void'.
  Types of parameters 'res' and 'value' are incompatible.
    The 'Object' type is assignable to very few other types. Did you mean to use the 'any' type instead?
      Type 'Object' is missing the following properties from type 'UserModel': email, image, name, roles, and 2 more.

![image](https://user-images.githubusercontent.com/41632489/214615068-41512e55-6451-4cf8-b37c-f3bde14bf2f2.png)

**Reply by andrebaltieri** [reply]
Bom dia, @MayconRDO, tenta fazer um cast do `UserModel` para `any` por favor:
`res: <any>UserModel`
